gear-programs/historical-proxy/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+.binpath
```

Comment:
do we really need that file? I believe the one from the project root should be fine

Reply:
Will be removed